### PR TITLE
Support for 'health' theme within runner 

### PR DESCRIPTION
**Pull request by Yuyuutsu**
### What is the context of this PR?
For PHM, we will be using the `health` theme given it is a health surveys. 
- Changed health theme from using default `SurveyConfig` class to `SocialSurveyConfig`
- Added a new test schema called `test_theme_health`
- Added unit tests which asserts that health theme instantiates a `SocialSurveyConfig` instance

### How to review 
- Test the `test_theme_health` schema in runner 
- `make test` and `make test-unit` pass

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by Yuyuutsu** [comment]
[Support for 'health' theme within runner (1 SP)](https://trello.com/c/rUb0gVsx/5447-support-for-health-theme-within-runner-1-sp)

File: tests/app/helpers/test_template_helpers.py [hunk 1]
```diff
@@ -127,6 +127,26 @@ def test_footer_warning_not_in_context_census_theme(app: Flask):
...
+        (
+            SurveyType.HEALTH,
+            "Test",
+            SocialSurveyConfig(),
+            {
+                "orgLogo": "ons-logo-en",
+                "orgLogoAlt": "Office for National Statistics logo",
+                "title": "Test",
+            },
+        ),
```
Comment (MebinAbraham) [inline tests/app/helpers/test_template_helpers.py:149]: I wouldn't bother with this. For now given health uses the `SocialSurveyConfig`, just have a single test that ensures when initialising a health theme, its instance is a `SocialSurveyConfig` instance (use the `isinstance` check).

Comment (Yuyuutsu) [reply tests/app/helpers/test_template_helpers.py:149]: Would `test_get_survey_config` (line 590) check if it is an instance of `SocialSurveyConfig` if I change the value of the parameter to `(SurveyType.HEALTH, "en", SocialSurveyConfig)`

Comment (MebinAbraham) [reply tests/app/helpers/test_template_helpers.py:149]: Ya, HEALTH is already defined for that test, which should currently fail. That would just need updating.

Comment (petechd) [reply tests/app/helpers/test_template_helpers.py]: OK, added what you've wanted now! Thanks

File: schemas/test/en/test_theme_health.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
```
Comment (berroar) [inline schemas/test/en/test_theme_health.json:22]: Minor but we should prob use social claims for this schema, like `test_theme_social`  (i know questionnaire_id will be renamed soon) e.g. 
```suggestion
        {
            "name": "case_ref",
            "type": "string"
        },
        {
            "name": "questionnaire_id",
            "type": "string"
        },
```

**Review by berroar — APPROVED** [review]

File: tests/app/helpers/test_template_helpers.py [hunk 3]
```diff
@@ -813,3 +813,14 @@ def test_include_csrf_token(app: Flask, include_csrf_token: bool):
...
+
+def test_health_survey_config(app: Flask):
+    with app.app_context():
+        survey_config = get_survey_config(
+            theme=SurveyType.HEALTH,
+            schema=QuestionnaireSchema({}),
+            language="en",
+        )
+
+    assert isinstance(survey_config, SocialSurveyConfig)
```
Comment (MebinAbraham) [inline tests/app/helpers/test_template_helpers.py:826]: This isn't needed. A test to check in instance already existing. You just need to update it https://github.com/ONSdigital/eq-questionnaire-runner/blob/support-health-theme/tests/app/helpers/test_template_helpers.py#L579-L580

Comment (petechd) [reply tests/app/helpers/test_template_helpers.py]: Oops, I didn't realise we check instance, updated tests now and removed the new one, thanks!

**Review by MebinAbraham — APPROVED** [review]

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Actually we need a change

**Review by MebinAbraham — APPROVED** [review]
Fine for now actually as our runner default is BUSINESS.

**Pull request merged**